src/views/states/list/StatesList.tsx

```diff
@@ -101,22 +101,30 @@ const StatesList: FC = () => {
             />
           </div>
 
-          <Table
-            cells={activeColumns}
-            rows={paginatedData}
-            gridBreakPoint={TableGridBreakpoint.none}
-            role="presentation"
-          >
-            <TableHeader />
-            {paginatedData.map((nns, index) => (
-              <StateRow
-                key={nns?.metadata?.name}
-                obj={nns}
-                activeColumnIDs={new Set(activeColumns.map(({ id }) => id))}
-                rowData={{ rowIndex: index, selectedFilters }}
-              />
-            ))}
-          </Table>
+          {filteredData.length > 0 ? (
+            <Table
+              cells={activeColumns}
+              rows={paginatedData}
+              gridBreakPoint={TableGridBreakpoint.none}
+              role="presentation"
+            >
+              <TableHeader />
+              {paginatedData.map((nns, index) => (
+                <StateRow
+                  key={nns?.metadata?.name}
+                  obj={nns}
+                  activeColumnIDs={new Set(activeColumns.map(({ id }) => id))}
+                  rowData={{ rowIndex: index, selectedFilters }}
+                />
+              ))}
+            </Table>
+          ) : (
+            <div className="cos-status-box">
+              <div data-test="empty-message" className="pf-u-text-align-center">
```

Comment:
Wouldn't be better to use `EmptyState` Patternfly React component instead of regular divs?